server/jetstream_sourcing_scaling_test.go

```diff
@@ -14,259 +14,126 @@
 package server
 
 import (
-	"encoding/json"
 	"fmt"
+	"github.com/nats-io/nats.go"
```

Comment:
I know a PITA but let's put proper tab indentation on these configs.

Reply:
? it is with a tab indentation for me. 
(I use gofmt to reformat)

Reply:
Manually put tabs in front of lines on configs, like we do in other tests etc. So reads easier, parser will handle.

Reply:
Sorry mis-understood which part of the code the comment was for, added tabs to the configs.